tests/test_amp/test_naive_bf16.py

```diff
@@ -0,0 +1,93 @@
+import torch
+import colossalai
+import torch.multiprocessing as mp
+from colossalai.amp import convert_to_naive_amp
+from tests.components_to_test.registry import non_distributed_component_funcs
+from colossalai.testing import assert_close_loose, rerun_if_address_is_in_use
+from colossalai.utils import free_port
+
+
+import copy
+import pytest
+from functools import partial
+
+
+def check_equal(a, b):
+    """
+    This function checks if two tensors are equal within tolerance
+    """
+    assert torch.allclose(a.float(), b.float(), rtol=1e-4, atol=1e-3), f'a = {a}, b = {b}'
+
+
+def run_naive_bf16_amp():
+    """
+    In this test, we compare the naive fp16 optimizer implemented in colossalai 
+    and fp32 torch optimizer
+    """
+
+    torch.backends.cudnn.benchmark = False
+    torch.backends.cudnn.deterministic = True
+
+    # create layer
+    test_models = ['repeated_computed_layers', 'nested_model', 'resnet18']
+    for test_name in test_models:
+        get_component_func = non_distributed_component_funcs.get_callable(test_name)
+        model_builder, train_dataloader, _, optim_class, _ = get_component_func()
+
+        # create model
+        naive_fp16_amp_model = model_builder(checkpoint=True).cuda()
+        naive_bf16_amp_model = copy.deepcopy(naive_fp16_amp_model).cuda().bfloat16()
```

Comment:
naive_fp16_amp_model.clone()

Reply:
it refers to test_naive_fp16.py
and 'NetWithRepeatedlyComputedLayers' object has no attribute 'clone'